packages/frontend/src/features/common/styles/constans/Device.ts

```diff
@@ -4,6 +4,7 @@ const size = {
   mobileL: "425px",
   tablet: "768px",
   laptop: "1024px",
+  LAPTOP: "1280px",
```

Comment:
zmień nazewnictwo, bo jak juz piszesz z małych liter to nie twórz później z wielkich bo nie jest to czytelne.

Reply:
to jest całkowicie do zmiany ale jest straszny bałagan z tym, poprawie całe nazewnicto jak będę składał strone wtedy będe miał dostępne wszystkie komponenty